packages/viewer/src/components/viewer-transform-widget/viewer-transform-widget.tsx

```diff
@@ -202,6 +222,47 @@ export class ViewerTransformWidget {
     }
   }
 
+  /**
+   * @ignore
+   */
+  @Watch('xTranslationDisabled')
+  @Watch('yTranslationDisabled')
+  @Watch('zTranslationDisabled')
+  @Watch('xRotationDisabled')
+  @Watch('yRotationDisabled')
+  @Watch('zRotationDisabled')
+  protected handleDisabledPropertyChanged(): void {
```

Comment:
I like that this is all in one method as opposed to six individual ones!

Reply:
Yeah, I was hoping the `@Watch` annotation supported a list, but it looks like they do not.
https://github.com/ionic-team/stencil/issues/1487